src/components/date_picker/super_date_picker/super_date_picker.tsx

```diff
@@ -575,3 +593,17 @@ export class OuiSuperDatePicker extends Component<
     );
   }
 }
+
+// @internal
+export type OuiCompressedSuperDatePickerProps = Omit<
+  OuiSuperDatePickerProps,
+  'compressed'
+>;
+
+// @internal
```

Comment:
Does this get stripped from output? Or what does this do?

Reply:
Its only purpose is to document that this is not part of the public API and for public consumption. It allows us to remove them without calling it a breaking change.